debian/changelog

```diff
@@ -162,4 +162,4 @@ webapp-manager (1.0.1) ulyana; urgency=medium
   * Initial release
 
  -- Clement Lefebvre <root@linuxmint.com>  Sat, 15 Aug 2020 10:24:00 +0100
-
+ 
```

Comment:
I guess you didn't mean to edit this

Reply:
Thank you for the suggestion. This is my first PR to a public repository. While reviewing the code, I understood that those edits were due to some lines added near the top of the code. Also, I am not so familiar with the intermediate to advanced git commands. It would be very much helpful if you could refer some more tutorials. Thanks in advance.

Reply:
Sure. Actually, there are plenty of tutorials and articles on git on the internet. If you have time and want to study git deeply, learn how to use its tools and (optionally) understand its internals, I would recommend reading [the Git book](https://git-scm.com/book/en/v2), which is also available in some languages other than English.

If you would rather learn on the go, you can just google "how to do X in git" every time you feel like there **could** be the ability to do `X` in git, and you very likely **will** find a solution, unless what you want to do is an anti-pattern, because git is really feature-rich. For the current PR, I would start with searching for "git amend commit", "git squash commits into one" and "git force-push". Just google these and read about the corresponding operations on whichever websites you like.

Good luck!

Reply:
Thank you very much for the suggestions. Actually that's what I do, every time I face a problem not only in `git` but also for any problem in `linux`. :)
I would surely go through the `the Git book` once I get time.